src/service/service.ts

```diff
@@ -7,14 +7,16 @@ type Options = {
 }
 
 class Service {
-    private client
+    // api gives access to low level gRPC calls.
+    api
+
     private token: string
     private mesgConfig: any
```

Comment:
```suggestion
    private serviceDefinition: any
```

I think `mesgConfig` is not the best name. I suggest `serviceDefinition`.
If you accept, let's do the change in a new PR.